### PR TITLE
WIA Polish

### LINKED ISSUE #2594: WIA: transcription button confusion
- The selected transcribed value conflicts with the next action green button and should be styled differently.
- When completing transcription, the two green done buttons next to each other makes it unclear which button should be selected.

### PR DESCRIPTION
**Pull request by beausmith**
## Overview
Resolves #2594 and more from the [spec doc](https://docs.google.com/document/d/12e_6sW2eutcJjSudFEZXORvnqVtU93DrV3PYbFqSgVs/edit#heading=h.n91har8oeghu).

## Demo Video or Screenshot
https://user-images.githubusercontent.com/28444/195442548-ebeb93cf-4f7b-4a22-ba72-53bde2034f31.mov

### CODE REVIEW

**Review by mattroe — COMMENTED** [review]
Interface and experience LGTM!

File: frontends/election-manager/src/screens/write_ins_transcription_screen.tsx [hunk 1]
```diff
@@ -295,7 +295,7 @@ export function WriteInsTranscriptionScreen({
               <HorizontalRule color="#cccccc" />
               <p>
                 {showNewTranscriptionForm ? (
-                  <InlineForm>
+                  <InlineForm as="span">
```
Comment (adghayes) [inline frontends/election-manager/src/screens/write_ins_transcription_screen.tsx:314]: Does this have a UI impact or does it just fix the console warning I've been seeing?

File: libs/ui/src/button.test.tsx [hunk 2]
```diff
@@ -25,39 +25,61 @@ describe('renders Button', () => {
...
+    render(
       <Button onPress={jest.fn()} primary noFocus>
         Primary
       </Button>
     );
-    expect(container.firstChild).toMatchSnapshot();
+    const button = screen.getByText('Primary');
+    expect(button).toHaveStyleRule('background', 'rgb(71,167,75)');
+    expect(button).toHaveStyleRule('color', '#FFFFFF');
+    expect(button).not.toHaveStyleRule('outline');
```
Comment (adghayes) [inline libs/ui/src/button.test.tsx:36]: I appreciate the moving away from snapshots.

File: frontends/election-manager/src/screens/write_ins_transcription_screen.tsx [hunk 3]
```diff
@@ -237,62 +257,58 @@ export function WriteInsTranscriptionScreen({
...
-                    </Button>
-                  ))}
-                </TranscribedButtons>
+                <React.Fragment>
+                  <TranscribedButtons>
+                    {[...cannedTranscriptions].map((val, i) => (
+                      <Button
+                        key={val}
+                        ref={i === 0 ? firstTranscriptionButton : undefined}
+                        primaryBlue={val === currentTranscribedValue}
```
Comment (adghayes) [inline frontends/election-manager/src/screens/write_ins_transcription_screen.tsx:302]: I like the blue.

**Review by adghayes — APPROVED** [review]
LGTM!

File: frontends/election-manager/src/screens/write_ins_transcription_screen.tsx (hunk 1 above)
Comment (eventualbuddha) [reply frontends/election-manager/src/screens/write_ins_transcription_screen.tsx:314]: Thank you! This has been bugging me.

**Pull request merged**